sky/clouds/service_catalog/data_fetchers/fetch_vast.py

```diff
@@ -0,0 +1,67 @@
+"""A script that generates the Vast Cloud catalog. """
+
+# pylint: disable=assignment-from-no-return
+import csv
+import json
+import re
+import sys
+
+from vastai_sdk import VastAI
```

Comment:
Since you've brought this up a second time, this is clearly a hard requirement that blocks a merge. Maybe we should update the google doc. 

I'll make sure this is done.

Reply:
https://docs.google.com/document/d/1oWox3qb3Kz3wXXSGg9ZJWwijoa99a3PIQUHBR8UgEGs/edit?usp=sharing

Reply:
here you go c5f75bcf5a3c128c8f00c5113bbe888aa4e1a8cb